controllers/clusteroperator_test.go

```diff
@@ -192,6 +186,40 @@ func normalizeTransitionTimes(got, expected osconfigv1.ClusterOperatorStatus) {
 	}
 }
 
+// getStatusDiff this is the same as v1helpers.GetStatusDiff except for:
```

Comment:
We don't need the behaviors that are removed in this version?

Reply:
we could add back relatedobjects and version when we add unit tests for them, but i am not using them .

Reply:
I have updated the name and comment to better indicate what it does